.env

```diff
@@ -0,0 +1,27 @@
+REACT_APP_CLIENT_PASSPHRASE=aaa
+REACT_APP_CLIENT_PUBLIC_KEY=bbb
+REACT_APP_CLIENT_PRIVATE_KEY=ccc
+REACT_APP_WALLET_APP_PUBLIC_KEY=ddd
+REACT_APP_WALLET_APP_AAT_SIGNATURE=ffff
+REACT_APP_POKT_USD_VALUE=0.14
+REACT_APP_SECURE_LS_ENCRYPTION_SECRET=v3rys3cr3tt0ken
+REACT_APP_SECURE_LS_ENCODING_TYPE=aes
+REACT_APP_SECURE_LS_IS_COMPRESSION=true
+REACT_APP_DISPATCHERS=https://node1.mainnet.pokt.network,https://node2.mainnet.pokt.network
+REACT_APP_HTTP_PROVIDER=
+REACT_APP_AAT_VERSION=1.0
+REACT_APP_MAX_DISPATCHERS=1
+REACT_APP_PROVIDER_TYPE=???
+REACT_APP_BLOCK_EXPLORER_BASE_URL=https://explorer.pokt.network
+REACT_APP_DASHBOARD_BASE_URL=https://mainnet.dashboard.pokt.network
+REACT_APP_BUY_POKT_BASE_URL=https://forum.pokt.network/t/secondary-markets-for-pokt/629
+REACT_APP_CHAIN=0001
+REACT_APP_CHAIN_ID=mainnet
+REACT_APP_BLOCK_TIME=900000
+REACT_APP_MAX_TRANSACTION_LIST_COUNT=100
+REACT_APP_TX_FEE=10000
+REACT_APP_SESSION_LENGTH=30
+REACT_APP_GATEWAY_BASE_URL="http://gateway.pokt.network/v1/lb/60a2ac11b1747c6552385c61"
+REACT_APP_HTTP_TIMEOUT=0
+REACT_APP_HTTP_HEADERS='{"Content-Type": "application/json", "Blockchain-Subdomain": "mainnet"}'
```

Comment:
blockchain-subdomain has been removed, please use actual subdomain

Reply:
this header may be blocked by CORS and throw an error

Reply:
sounds good, this was a hack for local dev thanks for catching

Reply:
actually this is double catch .env should not be committed so 👍🏻

Reply:
and in workflows env variables there is no blockchain subdomain